tests/performance/src/main/java/org/mule/runtime/core/processor/strategy/ProcessingStrategyBenchmark.java

```diff
@@ -0,0 +1,202 @@
+import static reactor.core.publisher.FluxSink.OverflowStrategy.ERROR;
+
+import org.mule.AbstractBenchmark;
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.construct.Flow;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.processor.ReactiveProcessor;
+import org.mule.runtime.core.api.processor.Sink;
+import org.mule.runtime.core.api.processor.strategy.ProcessingStrategy;
+import org.mule.runtime.core.internal.processor.strategy.DirectProcessingStrategyFactory;
+import org.mule.runtime.core.internal.processor.strategy.TransactionAwareProactorStreamEmitterProcessingStrategyFactory;
+import org.mule.runtime.core.internal.processor.strategy.TransactionAwareProactorStreamWorkQueueProcessingStrategyFactory;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.infra.Blackhole;
+import org.reactivestreams.Publisher;
+
+import java.util.function.Function;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.FluxSink;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.MonoSink;
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.Throughput)
+public class ProcessingStrategyBenchmark extends AbstractBenchmark {
+
+  private MuleContext muleContext;
+
+  private ProcessingStrategy directPs;
+  private ProcessingStrategy emitterPs;
+  private ProcessingStrategy workQueuePs;
+
+  private Flow flow;
+
+  private Sink directSink;
+  private Sink emitterSink;
+  private Sink workQueueSink;
+
+  private FluxSink<CoreEvent> directPipeline;
+  private FluxSink<CoreEvent> emitterPipeline;
+  private FluxSink<CoreEvent> workQueuePipeline;
+
+  private FluxSink<CoreEvent> directProcessor;
+  private FluxSink<CoreEvent> emitterProcessor;
+  private FluxSink<CoreEvent> workQueueProcessor;
+
+  private Sink directAllSink;
+  private Sink emitterAllSink;
+  private Sink workQueueAllSink;
+
+  @Setup(Level.Trial)
+  public void setUp() throws MuleException {
+    muleContext = createMuleContextWithServices();
+
+    directPs = new DirectProcessingStrategyFactory().create(muleContext, "direct_mb");
+    startIfNeeded(directPs);
+    emitterPs = new TransactionAwareProactorStreamEmitterProcessingStrategyFactory().create(muleContext, "emitter_mb");
+    startIfNeeded(emitterPs);
+    workQueuePs = new TransactionAwareProactorStreamWorkQueueProcessingStrategyFactory().create(muleContext, "workQueue_mb");
+    startIfNeeded(workQueuePs);
+
+    flow = createFlow(muleContext);
+
+    final ReactiveProcessor processor = p -> Flux.from(p).doOnNext(e -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1602))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7657&issues=AWjJeTCLotIXnmsUTyuV&open=AWjJeTCLotIXnmsUTyuV)

[ISSUE_KEY]: AWjJeTCLotIXnmsUTyuV